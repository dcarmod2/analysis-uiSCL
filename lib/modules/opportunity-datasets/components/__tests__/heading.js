import React from 'react'

import Heading from '../heading'

describe('Opportunity Datasets > Components > Heading', () => {
  it('should render', () => {
    expect(<Heading />).toMatchSnapshot()
  })
})
