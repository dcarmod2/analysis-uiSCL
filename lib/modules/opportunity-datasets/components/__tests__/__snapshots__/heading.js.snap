// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Opportunity Datasets > Components > Heading should render 1`] = `<ODHeading />`;
